Makefile

```diff
@@ -1,6 +1,8 @@
-
-# Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+RELEASE_REGISTRY?=ghcr.io/openfeature
+TAG?=latest
+RELEASE_IMAGE?=operator:$(TAG)
```

Comment:
I think it might be a _bit_ less confusing if we do something like:

```
RELEASE_IMAGE?=$(RELEASE_NAME):$(TAG)
```

Then I can control everything like:

```
make build-deploy-operator TAG=mytag RELEASE_REGISTRY=toddbaert RELEASE_NAME=myImgName
```

Which will pull `toddbaert/myImgName:mytag`

But this is just my opinion! I can go with the current way as well.

Reply:
sure, we can adapt it, thanks for the feedback :)

Reply:
adapted